src/components/Table/Table.test.jsx

```diff
@@ -464,79 +462,219 @@ describe('Table', () => {
     expect(wrapper.find('.bx--search-input').html()).toContain(`aria-hidden="true"`);
   });
 
-  it('cells should always wrap by default', () => {
```

Comment:
It's scary to me that tests are being removed.  Why not just keep the existing tests in place to catch regressions, and then add in the new tests?

Reply:
We are moving our tests away from enzyme and this test still exists but under another a slightly different name and refactored for the React Testing Library:

https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1645/files/073c843872db029503f1157850b83b5c63acbb3b#diff-f47894e32d4dbf684d2ac3f89df148c9R495-R500